shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -569,6 +569,10 @@ uintptr_t TizenRendererEvasGL::GetWindowId() {
       ecore_evas_ecore_evas_get(evas_object_evas_get(evas_window_)));
 }
 
+void* TizenRendererEvasGL::GetWindowHandle() {
+  return nullptr;
```

Comment:
Is this intentional or just unimplemented?

Reply:
It's half and half :smile: 
Evasgl version is not implemented yet.. so  This is for preventing tricky issues.